src/NzbDrone.Core/DecisionEngine/DownloadDecisionMaker.cs

```diff
@@ -64,14 +67,77 @@ private IEnumerable<DownloadDecision> GetMovieDecisions(List<ReleaseInfo> report
 
             foreach (var report in reports)
             {
+                //get wanted Language for this movie
+                var wantedLanguage = searchCriteria.Movie.Profile.Value.Language.ToString();
```

Comment:
Can you implement this in the Parser class? This really belongs there, since else, it won't work with already downloaded files.